.changeset/unlucky-files-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@animareflection/ui": minor
+---
+
+Add `Modal` component
```

Comment:
Duplicate changeset; I'm actually curious how Changesets handles this since it's exactly equal. Maybe let's run it through this PR and check on the release branch once they get compressed!